homeassistant/components/synology_srm/__init__.py

```diff
@@ -71,3 +76,9 @@ def get_srm_client_from_user_data(data) -> synology_srm.Client:
         client.http.disable_https_verify()
 
     return client
+
+
+def fetch_srm_device_id(client: SynologyClient):
```

Comment:
Why are you putting this function here since it's used only in the confug_flow ?

Reply:
I moved the function because it's always mocked in tests to I cannot have 100% coverage in this situation. It's also make sense in the end because I've another function just at the top to generate the client, so both functions handling the  `SynologyClient` are at the same place.